src/ansys/fluent/core/services/solution_variables.py

```diff
@@ -50,65 +50,73 @@ def get_zones_info(self, request):
         return self.__stub.GetZonesInfo(request, metadata=self.__metadata)
 
 
-class SVARInfo:
+class SolutionVariableInfo:
     """Provide access to Fluent SVARs and Zones information.
 
     Example
     -------
 
     .. code-block:: python
 
-        >>> svar_info = solver_session.svar_info
+        >>> solution_variable_info = solver_session.solution_variable_info
         >>>
-        >>> svars_info_wall_fluid = svar_info.get_svars_info(zone_names=['wall' , "fluid"], domain_name="mixture")
-        >>> svars_info_wall_fluid.svars
+        >>> solution_variables_info_wall_fluid = solution_variable_info.get_solution_variables_data(zone_names=['wall' , "fluid"], domain_name="mixture")
```

Comment:
`solution_variables_info_wall_fluid` is heavy and the original name was clumsy. We can fix into something terser and more meaningful: `wall_fluid_info`.

n.b. note the stray _s_ in your version.

Reply:
`get_svars_info` has transformed into `get_solution_variables_data`!

Reply:
> wall_fluid_info

ok. We will rename it to `wall_fluid_info`. Thank you.

Reply:
Done. Thank you.